### PR TITLE
Loosen Rails dependency for Rails 5.2

### PR DESCRIPTION
**Pull request by juliancheal**
Upped ruby version too, not sure if we want that in this PR.

/cc @jrafanie

### CODE REVIEW

File: manageiq-gems-pending.gemspec [hunk 1]
```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib", "lib/gems/pending", "lib/gems/pending/util"]
 
-  s.required_ruby_version = "> 2.2.2"
+  s.required_ruby_version = "> 2.3.1"
```
Comment (jrafanie) [inline manageiq-gems-pending.gemspec]: I wonder if this is should be 2.4?

https://www.ruby-lang.org/en/news/2019/03/31/support-of-ruby-2-3-has-ended/, at least for master...

Comment (jrafanie) [reply manageiq-gems-pending.gemspec]: @bdunne @Fryguy what do you think?

File: manageiq-gems-pending.gemspec [hunk 2]
```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |s|
...
 
-  s.add_runtime_dependency "actionpack",              ">= 5.0", "< 5.2"
-  s.add_runtime_dependency "activerecord",            ">= 5.0", "< 5.2"
-  s.add_runtime_dependency "activesupport",           ">= 5.0", "< 5.2"
+  s.add_runtime_dependency "actionpack",              "< 5.3"
+  s.add_runtime_dependency "activerecord",            "< 5.3"
+  s.add_runtime_dependency "activesupport",           "< 5.3"
```
Comment (jrafanie) [inline manageiq-gems-pending.gemspec]: LGTM here

Comment (jrafanie) [reply manageiq-gems-pending.gemspec]: curiosity: do we even need actionpack here?

```
05:11:49 ~/Code/manageiq-gems-pending (master) (2.4.6) + git grep -E "Action|Abstract"
lib/gems/pending/util/miq-exception.rb:  # Abstract MIQ Error Class
lib/gems/pending/util/miq-exception.rb:  # Concrete MIQ Policy Action Errors/Exceptions based on abstract Error class above
lib/gems/pending/util/miq-exception.rb:  class MiqActionError < Error; end
lib/gems/pending/util/miq-exception.rb:  class StopAction < MiqActionError; end
lib/gems/pending/util/miq-exception.rb:  class AbortAction < MiqActionError; end
lib/gems/pending/util/miq-exception.rb:  class UnknownActionRc < MiqActionError; end
lib/gems/pending/util/miq-exception.rb:  class PolicyPreventAction < MiqActionError; end
lib/gems/pending/util/win32/miq-powershell.rb:    #     $_ | Remove-Variable -Scope Local -ErrorAction SilentlyContinue
lib/gems/pending/util/win32/miq-psd.ps1:$ErrorActionPreference = "SilentlyContinue"
```

I think we pulled in it in 17196b6cbfa6de6681eb7769b68b251e9f5a2df3 , then 2b278f477a959c943ef6269e1eb93ea72596aa85, and finally f6704e20161a08f8a0683f98262290fb25a13e50

Comment (juliancheal) [reply manageiq-gems-pending.gemspec]: I don't know, shall I try removing it and see if it still works?

**Comment by juliancheal** [comment]
@miq-bot assgin @Fryguy

**Comment by juliancheal** [comment]
@miq-bot assign @Fryguy

**Pull request merged**